.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,45 @@
+name: "\U0001F41E Bug report"
+description: Create a report to help us improve
+labels:
+  - bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please carefully read the contribution docs before creating a bug report
+        👉 https://loopback.io/doc/en/contrib/Reporting-issues.html#loopback-4x-bugs
+
+        Please use the code sandbox template below to create a minimal reproduction
+        👉 https://codesandbox.io
```

Comment:
I think Code Sandbox may be a great way to make it easier to create and try out the minimal reproduction. However, this link doesn't point to a template, just the Code Sandbox homepage. Do we intend to create a template for others to create their sandboxes?

I think we should also still mention that simply creating a Git repository is still acceptable, as not everyone wants to learn and use a third-party service.

Reply:
What do you think of [this](https://githubbox.com/loopbackio/starter/tree/codesandbox)?

See [this](https://github.com/loopbackio/starter/tree/codesandbox) too